ironfish/src/rpc/routes/wallet/burnAsset.ts

```diff
@@ -71,6 +107,18 @@ routes.register<typeof BurnAssetRequestSchema, BurnAssetResponse>(
     const burn = transaction.burns[0]
 
     request.end({
+      asset: {
```

Comment:
I think we would want to have this be the final version of the API. Do we want each response to be nested like
```
{
  asset: {...}
}
```

This could make sense here since its the burnAsset endpoint and there could be other fields like `success` or something. If it were `getAsset` I would say we would not want the nesting though

Reply:
I agree we don't want nesting on getAsset but we do here